.eslintrc.js

```diff
@@ -10,5 +10,7 @@ module.exports = {
   ignorePatterns: ['temp', 'scip.ts', 'snapshots'],
   rules: {
     'no-sync': 'off',
+    'jsdoc/check-indentation': 'off',
+    'class-methods-use-this': 'error',
```

Comment:
Follow-up on [this comment](https://github.com/sourcegraph/scip-typescript/pull/192#discussion_r1006783990) by @varungandhi-src. [This rule](https://eslint.org/docs/latest/rules/class-methods-use-this) forbids class methods without instance context usage.

Reply:
This is really nice. Big +1 from me